index.js

```diff
@@ -54,7 +60,11 @@ async function fetchRepos() {
 		token
 	});
 
-	const repos = body.user.repositories.nodes;
+	const repos = body.user.repositories.nodes.map(repo => repo && ({
```

Comment:
I don't think you need the `repo &&`. If there are no repos, `.map()` will not run.